apps/src/Register3D.cpp

```diff
@@ -86,23 +86,58 @@ int main(int argc, char* argv[])
     Image8UC3::Pointer fixedImage;
     Image8UC3::Pointer movingImage;
 
+
+    /* PREVIOUS CODE
     // Read the OBJ file and static image
     io::OBJReader reader;
     reader.setPath(fixedPath);
     ITKMesh::Pointer origMesh;
     cv::Mat cvFixedImage;
     try {
         origMesh = reader.read();
-        cvFixedImage = reader.getTextureMat();
+        //cvFixedImage = reader.getTextureMat();
         fixedImage = OCVBridge::CVMatToITKImage<Image8UC3>(cvFixedImage);
     } catch (const std::exception& e) {
         std::cerr << e.what() << std::endl;
         return EXIT_FAILURE;
     }
+     */
+    io::OBJReader reader;
+    reader.setPath(fixedPath);
+    ITKMesh::Pointer origMesh;
+    cv::Mat cvFixedImage;
+
+
+
+    auto meshObj = rt::Data::Load(fixedPath);
+    cvFixedImage = meshObj->getImage();
+    origMesh = meshObj->getMesh();
```

Comment:
```suggestion
    auto meshObj = rt::Data::Load(fixedPath);
    auto cvFixedImage = meshObj->getImage();
    auto origMesh = meshObj->getMesh();
```

Reply:
Without the reader object, we need to have a getUVMap() function in the Mesh object. But we use a SpatialObject object in the Register3D.cpp file. So should I just add this function to SpatialObject and its subclasses? I think that's what we did for some other functions (like getMesh()), but it feels weird putting it in there if it is only needed for the Mesh subclass, you know?

Reply:
Also for this, I have started implementing that function, but was unsure what you wanted to return for objects that shouldn't have a UVMap. I was trying to return nullptr but realized the function doesn't return a raw pointer. Should I return an empty UVMap?

Reply:
In tandem with the other issue about removing the `return nullptr;`, I just took out the return statement and made these the same style as the function definitions for getMesh()

Reply:
I have set it to return an empty UVMap for now